app/templates/components/search-result.hbs

```diff
@@ -12,11 +12,11 @@
               {{#if index}}
                 >
               {{/if}}
-              {{{heading.value}}}
+              {{html-safe heading.value}}
```

Comment:
Not sure how you guys feel about adding new dependencies. Let me know if this should be changed.

Reply:
I'm okay with this.